ckan/lib/dictization/model_save.py

```diff
@@ -56,12 +57,16 @@ def resource_dict_save(res_dict, context):
                     obj.url_changed = True
             if key == 'url' and not new and obj.url != value:
                 obj.url_changed = True
+            if getattr(obj, key) != value:
```

Comment:
If the `key` was not present in the resource before this will raise an `AttributeError`.

Reply:
Sorry ignore me, I was reading the code wrong